app/controllers/users_controller.rb

```diff
@@ -44,6 +44,12 @@ def edit
   # PATCH/PUT /users/:id.:format
   def update
     set_tabs(params["user"]["tab"])
+
+    unless valid_filename?
+      render :edit, status: :bad_request
```

Comment:
maybe `:unprocessable_entity` here?

Reply:
I thought about that as well. But I wanted to keep it consistent with how the `update` action was already responding with errors:

https://github.com/thepracticaldev/dev.to/blob/e45363d63e0ed5a28db74b5dfe64c03d0a06c15b/app/controllers/users_controller.rb#L72